### PR TITLE
Deploy directly from backups folder

### PR DESCRIPTION
**Pull request by cdonati**

### CODE REVIEW

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/889/

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/893/

File: AppDB/backup/backup_recovery_helper.py [hunk 1]
```diff
@@ -311,35 +312,23 @@ def app_restore(storage, bucket_name=None):
...
+    logging.exception('Error while creating {}'.format(APP_TEMP_DIR_LOCATION))
+    return False
 
   # Deploy apps.
+  apps_to_deploy = [os.path.join(APP_TEMP_DIR_LOCATION, app) for app in
+    os.listdir(APP_TEMP_DIR_LOCATION)]
   if not deploy_apps(apps_to_deploy):
     logging.error("Failed to successfully deploy one or more of the "
       "following apps: {0}".format(apps_to_deploy))
-    if storage == StorageTypes.GCS:
```
Comment (menivaitsi) [inline AppDB/backup/backup_recovery_helper.py]: Make sure you revert this deletion and the one below.
If backed by GCS we shouldn't keep local backups.

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/894/

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/895/

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/896/

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/897/

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/898/

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/899/

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/900/

**Comment by menivaitsi** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/901/

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/903/

**Pull request merged**